src/soundcloud.js

```diff
@@ -0,0 +1,31 @@
+import { URL } from 'url';
+
+export const shouldTransform = string =>
+  new URL(string).host === 'soundcloud.com';
+
+const getSoundcloudIFrameSrc = string =>
+  `https://w.soundcloud.com/player` +
```

Comment:
Same

Reply:
Let's loose the `+` sings here too, like I suggested in https://github.com/MichaelDeBoey/gatsby-remark-embedder/pull/9#discussion_r324478757